src/components/Filters/components/FilterValue/FilterValue.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import {Text} from '@gravity-ui/uikit';
+import {block} from 'src/components/utils/cn';
+
+import {RemoveFilterButton} from '../RemoveFilterButton';
+
+import './FilterValue.scss';
+
+const b = block('filter-value');
+
+interface FilterValueProps {
+    className?: string;
+
+    onClick?: React.MouseEventHandler<HTMLButtonElement>;
+    onClear: VoidFunction;
+
+    fieldName: React.ReactNode;
+    fieldValue?: React.ReactNode;
+    emptyValueText?: React.ReactNode;
+
+    hideFieldName?: boolean;
+}
+
+export function FilterValue(props: FilterValueProps) {
```

Comment:
Why we do not use [Label](https://github.com/gravity-ui/uikit/tree/main/src/components/Label) here?

Reply:
We could use `Label` component here, but it does not work with keyboard well. It's not accessible via tab. I can implement `FilterValue` component with Label tag, but will keep `WrapperTag` as is

Reply:
I think we should use Label to have a consistent view. We can add some functionality to it if it needs

Reply:
Changed to Label and created RFC to discuss X icon